demo/src/main/java/com/orange/ods/demo/ui/components/chips/Chip.kt

```diff
@@ -103,6 +121,12 @@ fun ChipTypeDemo(chipType: ChipType, content: @Composable () -> Unit) {
     }
 }
 
+private fun Modifier.chipTypeSemantics(context: Context, focusedChipType: ChipType, selectedChipType: ChipType) = this.semantics {
+    stateDescription = if (selectedChipType == focusedChipType) {
+        "${context.getString(R.string.state_selected)}, ${context.getString(focusedChipType.nameRes)} ${context.getString(focusedChipType.descriptionRes)}"
```

Comment:
On my device there is no pause between vocalisations of `nameRes` and `descriptionRes`. It seems that separating strings with newline character do the trick:

```kotlin
"${context.getString(R.string.state_selected)}\n${context.getString(focusedChipType.nameRes)}\n${context.getString(focusedChipType.descriptionRes)}"
```